streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/master/coordinator/BaseCoordinator.java

```diff
@@ -0,0 +1,44 @@
+package io.ray.streaming.runtime.master.coordinator;
+
+import io.ray.api.Ray;
+import io.ray.streaming.runtime.master.JobMaster;
+import io.ray.streaming.runtime.master.context.JobMasterRuntimeContext;
+import io.ray.streaming.runtime.master.graphmanager.GraphManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public abstract class BaseCoordinator implements Runnable {
+
+  private static final Logger LOG = LoggerFactory.getLogger(BaseCoordinator.class);
```

Comment:
Better not to use `LOG` and `LOGGER` at the same time.

Reply:
I think is't OK, ray/java also use both of them.